database/sql/import-sql.sh

```diff
@@ -48,6 +48,10 @@ then
 	pre_dot=${file%%.sql}
 	# get rid of the ./
   db_name=${pre_dot##./}
+	if [[ True == $(get_config_value "sites.${db_name}.skip_provisioning" "False") ]]; then
```

Comment:
Wouldn't True need to be on quotes (since it's a string)?

Reply:
I copied that code from another provisioner.